### PR TITLE
Level 1 + Level 2 Post + Level 2 Like

### PR DESCRIPTION
**Pull request by gaia**

### CODE REVIEW

File: phases/phase-1/challenges/posts/gaia [hunk 1]
```diff
@@ -0,0 +1,2 @@
+B9FC92E2BD0695912F5C4B4174B554ED44F8150E672266C60FEC98A826FB0092
+E4B16D138C5A21BB7FCE53D8B14AAC9DDA61831BDD5A335D1B45897986262F7B
```
Comment (RiccardoM) [inline phases/phase-1/challenges/posts/gaia]: I'm sorry, but this looks like a `Like` transaction. 

Please remove this file from the file and add it to the proper one by running: 

```
echo E4B16D138C5A21BB7FCE53D8B14AAC9DDA61831BDD5A335D1B45897986262F7B > ./phases/phase-1/challenges/likes/gaia
```

**Comment by gaia** [comment]
Resubmitting as https://github.com/desmos-labs/primer/pull/51

**Comment by gaia** [comment]
Hi @RiccardoM. Why is Status:Waiting user applied to this?

Thanks

**Comment by RiccardoM** [comment]
@gaia Sorry, this was a leftover. In any case, this has been closed in favor of #51 as far as I've seen so no worries.

**Pull request closed without merging**